apps/etl-func/src/adapters/message-status.ts

```diff
@@ -0,0 +1,25 @@
+import {
+  MessageStatus,
+  MessageStatusEvent,
+  messageStatusEventSchema,
+} from "@/domain/message-status.js";
+
+/**
+ * Transform a MessageStatus into a MessageStatusEvent
+ **/
+
+export const getMessageStatusEventFromMessage = (
```

Comment:
```suggestion
export const getMessageStatusEventFromMessageStatus = (
```

Reply:
I would also move this function to the domain in the message-status-event file

Reply:
[f8df470](https://github.com/pagopa/io-messages/pull/62/commits/f8df47062c3fa0f4e331404ceb6e29fa81b8090f)